packages/loader/src/resource-deserialize/resources/parser/ReflectionParser.ts

```
@@ -21,7 +21,7 @@
     this.customParseComponentHandles[componentType] = handle;
   }
 
-  constructor(private readonly _context: ParserContext<IHierarchyFile, EngineObject>) {}
+  constructor(private readonly _context: ParserContext<IHierarchyFile, EngineObject>) { }
```

ESLint

[error] Delete `·`

(prettier/prettier)